tutorial/README.md

```diff
@@ -61,8 +61,9 @@ $ raw2ometiff n5_tile_directory/ LuCa-7color_Scan1.ome.tif --compression=zlib
 ```
 
 > NOTE: Viv currently uses [`geotiff.js`](https://geotiffjs.github.io/) for accessing data from remote TIFFs
-> over HTTP. There is limited compression support in this library, so the `--compression` flag must be specified with
-> either `zlib` or `Uncompressed` for viewing in the browser.
+> over HTTP and support the three lossless compression options supported
+> by `raw2ometiff` - `LZW`, `zlib`, and `Uncompressed`. `LZW` is the default if you
```

Comment:
Do you think that we should remove the explicit `--compression=zlib` flag from the instruction? I think it's fine as is, but would make the tutorial more simple.

Reply:
I think it's OK as-is because it shows that there is the option (without people having to look for it) and we are explicit that it is not necessary.

Reply:
Actually I will remove it since we explicitly say below that there is a `--compression` option, not just that you can change it.